unison-cli-integration/integration-tests/IntegrationTests/transcript.output.md

```diff
@@ -27,32 +33,30 @@ main = do
     _ -> ()
 ```
 
-``` ucm
+``` ucm :added-by-ucm
```

Comment:
Yeah, they show up after any Unison block, to hold the output of the implicit `load`.

`:added-by-ucm` now has a concrete behavior (whether applied to `ucm` or `unison`): the runner should discard that block without processing it when running, because a new version of that block will be created by the current run.